crates/aptos/src/common/init.rs

```diff
@@ -211,13 +215,28 @@ impl CliCommand<()> for InitTool {
                     "Account {} doesn't exist, creating it and funding it with {} Octas",
                     address, NUM_DEFAULT_OCTAS
                 );
-                fund_account(
+                let hashes = fund_account(
                     Url::parse(faucet_url)
                         .map_err(|err| CliError::UnableToParse("rest_url", err.to_string()))?,
                     NUM_DEFAULT_OCTAS,
                     address,
                 )
                 .await?;
+                let sys_time = SystemTime::now()
+                    .duration_since(SystemTime::UNIX_EPOCH)
+                    .map_err(|e| CliError::UnexpectedError(e.to_string()))?
+                    .as_secs()
+                    + 30;
+                for hash in hashes {
+                    client
+                        .wait_for_transaction_by_hash(
+                            hash.into(),
+                            sys_time,
+                            Some(Duration::from_secs(60)),
+                            None,
+                        )
+                        .await?;
+                }
```

Comment:
Can you just put this whole thing in one function, so the behavior is the same?

Reply:
Done done.